rest-api/etl/setup_local_database_for_etl.sh

```diff
@@ -54,7 +57,7 @@ set_local_db_connection_string
 function finish {  
   rm -f ${CREATE_DB_FILE}
   rm -rf ${CSV_DIR}
-  rm -f ${CREDS_FILE}
+  cleanup
```

Comment:
Per comment above: would clean up output here too if that's appropriate. (But OK if we need to persist for reuse.)

Reply:
Good call -- done.